R/format_data.R

```diff
@@ -135,6 +153,23 @@ fmt_number <- function(data,
     sep_mark <- ""
   }
 
+  # Get the `suffixing_inputs`
+  suffixing_inputs <- get_suffixing_inputs(suffixing)
+
+  # If choosing to perform large-number suffixing
+  # of numeric values, force `scale_by` to be 1.0
+  if (suffixing_inputs$to_suffix) {
+
+    if (!missing(scale_by) & !identical(scale_by, 1.0)) {
```

Comment:
The `&` should be `&&`. The single-`&` is only when you want to vectorize over whole vectors of logicals, not when you're trying to AND two single-element logical vectors.

Reply:
This has been corrected.